thruster/examples/static_file/main.rs

```diff
@@ -33,8 +33,8 @@ fn main() {
     info!("Starting server...");
 
     let app = App::<HyperRequest, Ctx, ()>::create(generate_context, ())
-        .get("/", m!(Ctx, [index]))
-        .get("/*", m!(Ctx, [file]));
+        .get("/", m![index])
+        .get("/*", m![file]);
```

Comment:
What is the difference between this and the `set404` function?

for example: 
https://github.com/thruster-rs/Thruster/blob/b218b8f7bb13d6d041bdb4608ff29d1f2ac3abee/thruster/examples/error_handling.rs#L99-L102

Reply:
great question!! The above code just sets it for the `GET` verb, `set404` does it for _all_ verbs. Check out here: https://github.com/thruster-rs/Thruster/blob/master/thruster/src/app/thruster_app.rs#L206-L220